vue/components/ui/molecules/form-input/form-input.vue

```diff
@@ -50,6 +75,15 @@
     line-height: 0px;
 }
 
+.form-input.inline .content {

```

Comment:
`.form-input.form-input-inline`. We usually use this pattern for this kind of stuff (e.g.: `.button-color.button-color-black`).